### DISCUSSION TITLE
React native sdk is not working on android

### DISCUSSION THREAD

**Discussion by ruymaster** [Web3Auth/Web3Auth#925] [Help]
I am trying to use RN sdk for android. Building apk is successful. then when running app on android, doesn't return info from google login. web browser for openlogin is correctly opened. then user info should be returned into app code. But app doesn't receive any info from login. How can I fix this?
returned error is like this.
```
[Error: The WebBrowser auth session is in an invalid state with no redirect handler when one should be set]
```
redirect url is input correctly, and it works well on iOS.

**Comment by shahbaz17** [comment]
Hey @ruymaster 

Please update the packages to the latest and take a look at the examples https://github.com/Web3Auth/examples/tree/main/react-native

**Comment by ruymaster** [comment]
Hello, @shahbaz17, Thanks for your help!
